lib/JMAP/Tester.pm

```diff
@@ -205,7 +209,7 @@ L<JMAP::Tester::Response> objects.
 =cut
 
 sub request {
-  my ($self, $calls) = @_;
+  my ($self, $input_request, $arg) = @_;
```

Comment:
`$arg` is unused

Reply:
Yeah, I meant to add a means to request/demand wrapping or not.  I'll just drop it until then.